composer/algorithms/channels_last/channels_last.py

```diff
@@ -25,9 +27,24 @@ def apply_channels_last(model: torch.nn.Module) -> None:
 
 
 class ChannelsLast(Algorithm):
-    """Changes the memory format of the model to ``torch.channels_last``. This usually yields improved GPU utilization.
+    """Changes the memory format of the model to ``torch.channels_last``. This usually improves GPU utilization.
```

Comment:
Link to official channels last docs here and on line 19 (`apply_channels_last` docstring)